src/scene/shader-lib/programs/lit-shader-options.js

```diff
@@ -252,11 +252,11 @@ class LitShaderOptions {
     toneMap = -1;
 
     /**
-     * One of "envAtlasHQ", "envAtlas", "cubeMap", "sphereMap".
+     * One of REFLECTIONSRC_*** constants.
      *
      * @type {string}
      */
-    reflectionSource = null;
+    reflectionSource = REFLECTIONSRC_NONE;
```

Comment:
I prefer to avoid introducing new public API constants unless there's a really good reason. Why not just tighten up the type to:

```
* @type {"envAtlasHQ" | "envAtlas" | "cubeMap" | "sphereMap"}
```

Reply:
those constants are already in the public API here: https://api.playcanvas.com/classes/Engine.LitShaderOptions.html#reflectionsource

and for consistency, I'm turning those into enums as all other similar constants, instead of just strings.

Are you suggesting we change this and avoid new constants in preference to strings?

Reply:
Definitely worth a discussion. Ideally we keep the codebase consistent, instead of the mix of two styles.